src/VisualStudioExtension/QsharpVSIX/QsLanguageClient.cs

```diff
@@ -134,5 +140,24 @@ public void OnTelemetry(JToken args)
                 { Debug.Assert(false, $"error sending telemetry: \n{ex}"); }
             }
         }
+
+        private static string GetVisualStudioVersion()
```

Comment:
Seems a bit hacky. I think it should be possible to go via DTE.Version without load issues.

Reply:
Definitely agree that it's hacky, but when I tried `DTE.Version` it only gave me "16.0" (no minor version). :(

Reply:
Arg, is that annoying. I didn't find a better solution either. Ok, hacky it is then.